assets/js/components/KeyMetrics/MetricTileText.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * MetricTileText component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import ChangeBadge from '../ChangeBadge';
+import PreviewBlock from '../PreviewBlock';
+
+export default function MetricTileText( {
+	Widget,
+	loading,
+	title,
+	metricValue,
+	subText,
+	previousValue,
+	currentValue,
+} ) {
+	if ( loading ) {
+		return (
+			<Widget noPadding>
+				<PreviewBlock width="100%" height="142px" padding />
+			</Widget>
+		);
+	}
+
+	return (
+		<Widget noPadding>
+			<div className="googlesitekit-km-widget-tile googlesitekit-km-widget-tile--text">
+				<h3 className="googlesitekit-km-widget-tile__title">
+					{ title }
+				</h3>
+				<div className="googlesitekit-km-widget-tile__body">
+					<div className="googlesitekit-km-widget-tile__metric">
+						{ metricValue }
+					</div>
+					<p className="googlesitekit-km-widget-tile__subtext">
+						{ subText }
+					</p>
+					<div className="googlesitekit-km-widget-tile__metric-change-container">
+						<ChangeBadge
+							previousValue={ previousValue }
+							currentValue={ currentValue }
+						/>
```

Comment:
We need to add the `isAbsolute` property to the `ChangeBadge` element to let us use relative values. See how we do it in the `MetricTileNumeric` component: https://github.com/google/site-kit-wp/blob/8fe4cfb0510912af79ff7604b1bd11f717cc5703/assets/js/components/KeyMetrics/MetricTileNumeric.js#L67

The problem here is that in the current implementation (without the property), the `ChangeBadge` will calculate the relative difference between previous and current values, but we need the absolute difference. In other words, if the current value is `50%` and the previous value is `25%`, the badge will show `+50%` which is incorrect because the top traffic source increased by `25%` from previous week in absolute values and that is what we need to show to users to make them easier to understand the difference between last week and this week.

See how the LoyalVisitors widget is implemented as an example for more details.